core/types/transaction_conditional_test.go

```diff
@@ -37,26 +37,26 @@ func TestTransactionConditionalCost(t *testing.T) {
 		{
 			name: "default cost per account",
 			cond: TransactionConditional{KnownAccounts: map[common.Address]KnownAccount{
-				common.Address{19: 1}: KnownAccount{},
-				common.Address{19: 2}: KnownAccount{},
+				{19: 1}: {},
```

Comment:
This is unnecessary diff that adds noise for reviewers, its not related to the feature in this PR so we should try to reduce diff in this PR

Reply:
I was meaning to get rid of the pesky vscode warnings about dead code, I do see your point that this is noise. Happy to revert this change from this PR and create a separate one to address this cosmetic issue.